test/fixtures/raw-loader.js

```diff
@@ -1,5 +1,5 @@
 exports.__es6Module = true;
 exports.default = function(source) {
-	return new Buffer(source.toString("hex") + source.toString("utf-8"), "utf-8");
+	return Buffer.from(source.toString("hex") + source.toString("utf-8"), "utf-8");
```

Comment:
So even though this will give you eslint warnings, leave this as `new Buffer`, and then add 
`// eslint-disable-line` as an inline comment to disable this warning.